### DISCUSSION TITLE
Low-latency surface kernel

### DISCUSSION THREAD

**Discussion by citadel-lewis** [linux-surface/linux-surface#958] [Ideas]
I'm wondering if it's possible to add a low-latency version of the surface kernel to the ubuntu repository? I've been trying to apply the surface patches to kernel sources and build a low-latency version myself but it keeps failing. I need low-latency kernel for audio production.

**Comment by qzed** [comment]
I don't have any plans to maintain an extra set of patches for this at the moment, sorry. Depending on the device, however, you might get quite far with the recent kernels, so you could give Ubuntu's latest real-time kernel a try.
